### ISSUE TITLE
Multiplication between RGB types seems to be missing

### ISSUE THREAD

**Issue by anarnold** [JuliaGraphics/ColorVectorSpace.jl#119]
```julia
julia> RGB{Float64}(1, 1, 1) * RGB{Float64}(1, 1, 1)
ERROR: MethodError: no method matching *(::RGB{Float64}, ::RGB{Float64})
Closest candidates are:
  *(::Any, ::Any, ::Any, ::Any...) at operators.jl:529
  *(::Real, ::AbstractRGB) at /home/afertin/.julia/packages/ColorVectorSpace/wI546/src/ColorVectorSpace.jl:123
  *(::AbstractRGB, ::Real) at /home/afertin/.julia/packages/ColorVectorSpace/wI546/src/ColorVectorSpace.jl:147
  ...
Stacktrace:
 [1] top-level scope at none:0
```

If I add:
```julia
import Base.*, ColorVectorSpace.color_rettype, ColorVectorSpace.multype
(*)(a::AbstractRGB{S}, b::AbstractRGB{T}) where {S,T} = color_rettype(a,b){multype(S,T)}(red(a)*red(b), green(a)*green(b), blue(a)*blue(b))
```
therefore, I can multiply two RGB images.

**Comment by timholy** [comment]
Why is channelwise the right definition of `rgb1*rgb2`? What if someone else thinks it should be the channelwise dot product? And someone else thinks it's the outer product,
```juila
[r1*r2 r1*g2 r1*b2
 g1*r2 g1*g2 g1*b2
 b1*r2 b1*g2 b1*b2]
```

**Comment by arnold-dev** [comment]
OK I understand the ambiguity here. I was surprised, because channelwise product is more common in image processing libraries. Sorry about that.

**Comment by timholy** [comment]
```julia
julia> using Colors

julia> chantimes(c1, c2) = mapc(*, c1, c2)
chantimes (generic function with 1 method)

julia> c1, c2 = rand(RGB{Float32}), rand(RGB{Float32})
(RGB{Float32}(0.5848855f0,0.29970312f0,0.60462904f0), RGB{Float32}(0.57155025f0,0.32285166f0,0.91198397f0))

julia> chantimes(c1, c2)
RGB{Float32}(0.33429143f0,0.09675965f0,0.551412f0)

julia> using TestImages

julia> img = testimage("lighthouse");

julia> chantimes.(img, img)
```
If this is not discoverable, a [democard](https://github.com/JuliaImages/juliaimages.github.io) would be wonderful!

**Comment by timholy** [comment]
Actually, it also occurs to me that we might have enough operators to distinguish this: `⋅` (`\cdot`), `*`, and `⊗` (`\otimes`). Assuming all these are broadcastable then perhaps we could define this operation. What do others think?

**Comment by johnnychen94** [comment]
> Assuming all these are broadcastable then perhaps we could define this operation. 

Great idea and I'd think it would be really useful! Although for me the only problem is that I can't type these symbols in vscode 😂

**Comment by timholy** [comment]
https://github.com/julia-vscode/julia-vscode/issues/613 suggests you might try this extension: https://marketplace.visualstudio.com/items?itemName=GuidoTapia2.unicode-math-vscode

**Comment by timholy** [comment]
Closed by #131
